functions/go/inflate-helm-chart/README.md

```diff
@@ -147,15 +147,15 @@ $ kpt pkg tree
 └── [service_test-minecraft.yaml]  Service test-minecraft
 ```
 
-### Example with kustomize
+### Example kustomization with remote chart
```

Comment:
This is inconsistent with other functions. I would suggest focusing on just the functionConfig here and usage with kpt like we do for other functions. Include an example to demonstrate how to use this function with kustomize ? We can follow the same model for other functions as well. So when we introduce changes in kustomize CLI, those will impact just the kustomize examples so keeping the function pages decoupled from kustomize's changes as well.

Reply:
I have moved the kustomize example to its own example page, decoupling it from the function page.

I can also remove the kustomize example entirely if it is not desired. The main reason for including it was to test `valuesInline`, as there is no way to test this with kpt CLI.

Reply:
I am happy with the new changes.